### DISCUSSION TITLE
RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%99%AE%E9%80%9A%E7%89%A9%E7%90%86%E5%AD%A6/14%20Lecture%2014/

### DISCUSSION THREAD

**Discussion by giscus** [RoderickShao/RoderickShao_notebook#18] [Announcements]
# RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%99%AE%E9%80%9A%E7%89%A9%E7%90%86%E5%AD%A6/14%20Lecture%2014/

RoderickShao's NoteBook

https://roderickshao.github.io/RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%99%AE%E9%80%9A%E7%89%A9%E7%90%86%E5%AD%A6/14%20Lecture%2014/

**Comment by Provence-24** [comment]
逻辑尝试梳理： 衍射
圆孔衍射 衍射因子
光栅 衍射因子+干涉因子 理解包络线
干涉因子=>半角宽
beta处引入常数m，得到光栅公式
求微分，得到分散本领
将半角宽（最小能分辨角度）代入得分辨本领

**Reply by Provence-24** [reply]
还有一个就是根据缺级来判断d和a的关系

**Comment by JasperZJU** [comment]
近场远场衍射可以通过 $FN = \frac{a^2}{4D\lambda}$ 来判断，FN<<1可以看作是弗朗禾费衍射，FN在1附近或者大于1可以看作是菲涅尔

**Reply by JasperZJU** [reply]
还有就是一般来说傅里叶变换的定义里面乘的是 e^(-jωt)，不过这边因为最后出来频域图像（衍射pattern）是对称的，所以最后结果没区别
